src/test/java/training/article/create.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Create an article
+
+  Background:
+    * url url
+    * def login = call read('../sign_in/sign_in.feature@loginSuccessfully')
+    * def username = username
+    * def token = "Bearer " + login.response.user.token
```

Comment:
think about this. This steps are in ALL the features. Look at the hooks section in karate documentation and try to don't repeate this code :)